tools/buildsteps/win32/make-mingwlibs.bat

```diff
@@ -4,22 +4,22 @@ SETLOCAL
 rem batch file to compile mingw libs via BuildSetup
 SET WORKDIR=%WORKSPACE%
 rem set M$ env
-call "%VS140COMNTOOLS%\..\..\VC\bin\vcvars32.bat" || exit /b 1
+call "%VS140COMNTOOLS%\..\..\VC\bin\amd64\vcvars64.bat" || exit /b 1
 
 SET PROMPTLEVEL=prompt
 SET BUILDMODE=clean
 SET opt=mintty
-SET build32=yes
-SET build64=no
+SET build32=no
+SET build64=yes
```

Comment:
please don't change default values, if you pass a build64 param then it should build x64 version of libs. also need to call to vcvarsXX.bat depending on build.